sites/svelte.dev/src/routes/(authed)/repl/+page.js

```diff
@@ -1,18 +1,36 @@
 import { redirect } from '@sveltejs/kit';
 
+/**
+ * create a query params string from an object of query parameters
+ * @param {Record<string, string>} queries
+ * @returns {string}
+ */
+function add_query_params(queries) {
```

Comment:
Why not use `new URLSearchParams().toString()`?

Reply:
It was done differently before and I hadn't thought of that...I'll fix it now

Reply:
Fixed it